### ISSUE TITLE
Tagline

### ISSUE THREAD

**Issue by MoOx** [postcss/postcss.com#46]
I think this should be discussed, following https://github.com/postcss/postcss.com/issues/4#issuecomment-170269433

**Comment by MoOx** [comment]
One other (simple) idea: Enhance your CSS.

**Comment by MoOx** [comment]
(reminder: most people don't write plugin and probably don't give a shit about the fact that it use js - most people use Sass these days, which is in ruby or c ^^)

**Comment by marcustisater** [comment]
I just don't see a reason for the entire PostCSS tagline to change, it has been sticking there for too long. However I am very open for ideas and a discussion about this, maybe it will change my mind.

**Comment by mxstbr** [comment]
I agree with @MoOx, I think the current tagline isn't very optimal. We need to sell the benefit of using PostCSS, not how the benefit is achieved!

**Comment by marcustisater** [comment]
Feel free to add more taglines other Open Source Project got, can get us thinking :thought_balloon:  

Sass tagline "CSS with superpowers"
Stylus tagline "Expressive, Dynamic, Robus CSS"

**Comment by marcustisater** [comment]
If we were to change the tagline I think we should adress PostCSS of being "universal" - Anything can be added - there is no limit or anything that can't be done.

**Comment by MoOx** [comment]
- "Choose your CSS flavor"
- "Choose your CSS superpowers" (and that's a nice reference, Sass should be flattered).
- "Make your own CSS flavor".

**Comment by ai** [comment]
We discussed before, that we should focused on content. Does this tagline is a userful for our users content?

I think we should create a text only if it will be design purpose.

**Comment by MoOx** [comment]
Good design make no sense with no real content. All good designers will tell you that.

I think it's really important to design the website by thinking to this things first.
A tagline is super important imo.

**Comment by ai** [comment]
@MoOx why you think tagline is important? Let’s define tagline task and we will find the best one.

**Comment by marcustisater** [comment]
Yeah but the tagline should really stick with the brand as well. We should not only think in terms of design.

:joy:
> "Choose your CSS superpowers" (and that's a nice reference, Sass should be flattered).

**Comment by ai** [comment]
> Yeah but the tagline should really stick with the brand as well

@marcustisater let’s focused on real task. What you want from tagline?

Without it we will create a “because every compnay need it” text without any benefits for us.

**Comment by ai** [comment]
For example, some companies think that tagline should show a emotional content of brand. The non-ration idea and spirit. If we agree with it, we should find this idea and then make it into a text.

Other companies think that tagline is a short description of idea. In this case we should make something like that main description in `package.json`.

**Comment by marcustisater** [comment]
Great point, I think we should focus on a tagline that is short and simple that's why I've personally liked "Transform you're CSS with JS Plugins" so much.

**Comment by peterramsing** [comment]
Some thoughts that came to mind:

I'd be curious what the target audience is. Is this speaking to PostCSS plugin developers or CSS developers looking for PostCSS Plugins? I glanced at #26 and this sort of relates to this issue as well.

*Some ideas:*

## To the PostCSS Plugin Developers:
1. "A tool for transforming styles with JS plugins." (from `package.json`)
1. "Enhancing CSS with the power of JavaScript."
1. "A pluggable platform to transform CSS with JavaScript."

## To the CSS Plugin Developers:
1. "Empowering the css developer."
1. "Empowering the css developer with a pluggable compilation tool."
1. "Your CSS with the power of JavaScript."
1. "Your CSS, Enhanced."
1. "The ultimate, and pluggable toolkit for the CSS developer"
1. "CSS empowered JavaScript."

## To both in one:
1. "A toolkit for enhancing and optimizing CSS with the power of pluggable JavaScript."
1. "A toolkit for CSS using the power of JavaScript."
1. "The ultimate pluggable toolkit for CSS with JavaScript."
1. "CSS enhanced with JavaScript by the ultimate pluggable toolkit."
1. "Enhancing CSS with a pluggable JavaScript toolkit."
1. "JavaScript turbocharging the vanilla CSS."

**Comment by marcustisater** [comment]
Good thoughts @peterramsing. I'll suggest we come up with a few more taglines then make a poll if we can't come to an agreement together. 

My favourite one so far:

> Your CSS, Enhanced

**Comment by ai** [comment]
@marcustisater let’s think rational first and use polls only if we will have few results with same rational results.

**Comment by ai** [comment]
I prefer taglines like “Your CSS with the power of JavaScript” rather than like “Your CSS, Enhanced”. Because second one do not contains any explanation. Every tool can tell it about itself. As result, users will not belive in this text.

**Comment by marcustisater** [comment]
That makes sense and I agree with you but does it attract users to PostCSS? People are using Sass now a days even though is made in Ruby and C++(LibSass). 

I am thinking of how to adopt new new-users to PostCSS - they might be thinking "Oh, sounds difficult", "Javascript with CSS, no thank you". Does the tagline "Your CSS with the power of JavaScript" really attracts users to start looking into PostCSS more? I don't know. This is sure a difficult one because both sides has negatives/positives sides.

**Comment by marcustisater** [comment]
as @peterramsing mentioned, maybe something in between. **Both in one**

**Comment by ai** [comment]
@marcustisater if you want to attract users, let’s make it as a main purpose for the tagline.

I think that it is immpossible to do something, if you didn’t understand why you need it.

This is why I afraid that we start this issue as cargo cult: we saw that big brands has tagline, so we try to find our. But it is a bad way. We should use rational way: understand what we need and how we can solve it.

**Comment by ai** [comment]
Yeap, I think too that we should create tagline for both type of users. Especially because many companies write custom plugins.

**Comment by ai** [comment]
My tagline version: “Basement to do anything with CSS”. It is a best explanation, but not from user perspective, so it is just a suggestion.

**Comment by marcustisater** [comment]
I understand you completely, the best way is to aim at both of the target audience imo. @stephenway wrote some very good questions at #26

**Comment by marcustisater** [comment]
This is a very difficult one and it's high prioritized issue to get moving, try promote this @ai if possible

**Comment by ai** [comment]
This task is not high important for me, because I think we should always see a real usage of any design item.

So it is too hard to create a tagline without a place, when it will be used.

So, when you plan to use it?

**Comment by marcustisater** [comment]
Sure, that's fair. I was just thinking to actually try set a tagline for designs to work with.

**Comment by giuseppeg** [comment]
*"A tool for transforming CSS with JavaScript"* is almost perfect in my opinion.

It is just missing one important piece that is *"plugins"* 
which communicates the most important concept behind PostCSS: modularity.

With such a small addition we have all we need to explain what PostCSS is in one line:

![g1456189085113493981](https://cloud.githubusercontent.com/assets/711311/13238119/a12ef5da-d985-11e5-83a4-ebca31e0db65.jpg)

**Comment by marcustisater** [comment]
Thanks for suggestion,

We should bring this topic up for some hot discussion again soon.
